frontend/packages/console-shared/src/components/alerts/AlertSeverityIcon.tsx

```diff
@@ -19,7 +19,7 @@ const AlertSeverityIcon: React.FC<AlertSeverityIconProps> = ({
             fontSize,
             color: 'var(--pf-global--danger-color--100)',
           }}
-          alt="Monitoring Alert"
+          aria-label="Monitoring Alert"
```

Comment:
I'm not sure this is right. Are these SVGs? I believe they should be `alt`.

cc @jessiehuff

Reply:
Truthfully, svgs are tough, and I've never seen a consensus on that debate between documentation and experts. One of the [best comparisons](https://www.deque.com/blog/creating-accessible-svgs/) I've seen is from deque (who created Axe) which essentially ends up stating that they found that one solution was not necessarily superior over the other. They cover all combinations of alt, roles, title/desc, and aria, and conclude: 

"On the flip side, there was no clear frontrunner for “best in show” for the different patterns when relaying basic alternative content. This is actually good news! Since many of the patterns were able to render the alt content, that equates to more pattern choices for designers and developers. For SVGs needing more descriptive alternative content, pattern 11 `<svg> + role="img" + <title> + <desc> + aria-labelledby="[ID]"` was the most reliable choice for the different browser and screen readers that were tested."

I've often still recommended alt because it's not just for screen readers, it's frequently used for low bandwidth users as well (when an image can't load, it will show the alt instead). I have read in some articles that svgs have limited support with alt or that it needs `role="img"` for it to be valid, but it looks like there's a lot of different ways for us to make this accessible. I think if it sounds good in testing, I'm ok with whatever solution we choose.

Some other notable documentation/articles that are interesting to include is this [comparison](https://www.scottohara.me/blog/2019/05/22/contextual-images-svgs-and-a11y.html) by Scott O'Hara, the more old school [recommendations](https://developer.paciellogroup.com/blog/2013/12/using-aria-enhance-svg-accessibility/) by Leonie Watson (both O'Hara and Watson are considered a11y experts), and the [Appendix H ](https://www.w3.org/TR/SVG11/access.html)on svg accessibility by W3.

Reply:
Thanks @jessiehuff. @jcaianirh I'd stick with the existing `alt` unless there is a specific reason we need to change. Does this cause problems with the new Patternfly version?

Reply:
@spadgett pf has been updated to support alt tags again. I will put this on hold and update the pr.